### PR TITLE
[doc] Add topic load latency metrics documentation

### PR DESCRIPTION
**Pull request by codelipenghui**
- Add `topic_load_times`
- Move the managed ledger cache and connection metrics under the broker metrics section

### Documentation

Check the box below or label this PR directly.

Need to update docs? 

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)
  
- [ ] `no-need-doc` 
(Please explain why)
  
- [x] `doc` 
(Your PR contains doc changes)

- [ ] `doc-added`
(Docs have been already added)

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@codelipenghui would you like to invite an engineer to review this PR from the technical perspective? Then writers can review it from the technical writing perspective, thanks.

@momo-jun could you please help review this PR after the technical review? Thanks

File: site2/docs/reference-metrics.md [hunk 1]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+| pulsar_ml_cache_used_size | Gauge  | The size in byte used to store the entries payloads |
+| pulsar_ml_count | Gauge  | The number of currently opened managed ledgers |
+| topic_load_times | Summary | The topic load latency calculated in milliseconds |
+| pulsar_active_connections| Gauge | The number of active connections |
+| pulsar_connection_created_total_count | Gauge | The total number of connections |
+| pulsar_connection_create_success_count | Gauge | The number of successfully created connections |
+| pulsar_connection_create_fail_count | Gauge | The number of failed connections |
+| pulsar_connection_closed_total_count | Gauge | The total number of closed connections |
+| pulsar_broker_throttled_connections | Gauge | The number of throttled connections |
+| pulsar_broker_throttled_connections_global_limit | Gauge | The number of throttled connections because of per-connection limit |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:146]:
```suggestion
| pulsar_broker_throttled_connections_global_limit | Gauge | The number of throttled connections due to per-connection limit. |
```

File: site2/docs/reference-metrics.md [hunk 2]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+| pulsar_ml_cache_pool_used | Gauge  | The total used memory of chunk lists in direct arena |
+| pulsar_ml_cache_used_size | Gauge  | The size in byte used to store the entries payloads |
+| pulsar_ml_count | Gauge  | The number of currently opened managed ledgers |
+| topic_load_times | Summary | The topic load latency calculated in milliseconds |
+| pulsar_active_connections| Gauge | The number of active connections |
+| pulsar_connection_created_total_count | Gauge | The total number of connections |
+| pulsar_connection_create_success_count | Gauge | The number of successfully created connections |
+| pulsar_connection_create_fail_count | Gauge | The number of failed connections |
+| pulsar_connection_closed_total_count | Gauge | The total number of closed connections |
+| pulsar_broker_throttled_connections | Gauge | The number of throttled connections |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:145]:
```suggestion
| pulsar_broker_throttled_connections | Gauge | The number of throttled connections. |
```

File: site2/docs/reference-metrics.md [hunk 3]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+| pulsar_ml_cache_pool_allocated | Gauge  | The total allocated memory of chunk lists in direct arena |
+| pulsar_ml_cache_pool_used | Gauge  | The total used memory of chunk lists in direct arena |
+| pulsar_ml_cache_used_size | Gauge  | The size in byte used to store the entries payloads |
+| pulsar_ml_count | Gauge  | The number of currently opened managed ledgers |
+| topic_load_times | Summary | The topic load latency calculated in milliseconds |
+| pulsar_active_connections| Gauge | The number of active connections |
+| pulsar_connection_created_total_count | Gauge | The total number of connections |
+| pulsar_connection_create_success_count | Gauge | The number of successfully created connections |
+| pulsar_connection_create_fail_count | Gauge | The number of failed connections |
+| pulsar_connection_closed_total_count | Gauge | The total number of closed connections |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:144]:
```suggestion
| pulsar_connection_closed_total_count | Gauge | The total number of closed connections. |
```

File: site2/docs/reference-metrics.md [hunk 4]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+| pulsar_ml_cache_pool_active_allocations_small | Gauge  | The number of currently active small allocations in direct arena |
+| pulsar_ml_cache_pool_allocated | Gauge  | The total allocated memory of chunk lists in direct arena |
+| pulsar_ml_cache_pool_used | Gauge  | The total used memory of chunk lists in direct arena |
+| pulsar_ml_cache_used_size | Gauge  | The size in byte used to store the entries payloads |
+| pulsar_ml_count | Gauge  | The number of currently opened managed ledgers |
+| topic_load_times | Summary | The topic load latency calculated in milliseconds |
+| pulsar_active_connections| Gauge | The number of active connections |
+| pulsar_connection_created_total_count | Gauge | The total number of connections |
+| pulsar_connection_create_success_count | Gauge | The number of successfully created connections |
+| pulsar_connection_create_fail_count | Gauge | The number of failed connections |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:143]:
```suggestion
| pulsar_connection_create_fail_count | Gauge | The number of failed connections. |
```

File: site2/docs/reference-metrics.md [hunk 5]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+| pulsar_ml_cache_pool_active_allocations_normal | Gauge  | The number of currently active normal allocations in direct arena |
+| pulsar_ml_cache_pool_active_allocations_small | Gauge  | The number of currently active small allocations in direct arena |
+| pulsar_ml_cache_pool_allocated | Gauge  | The total allocated memory of chunk lists in direct arena |
+| pulsar_ml_cache_pool_used | Gauge  | The total used memory of chunk lists in direct arena |
+| pulsar_ml_cache_used_size | Gauge  | The size in byte used to store the entries payloads |
+| pulsar_ml_count | Gauge  | The number of currently opened managed ledgers |
+| topic_load_times | Summary | The topic load latency calculated in milliseconds |
+| pulsar_active_connections| Gauge | The number of active connections |
+| pulsar_connection_created_total_count | Gauge | The total number of connections |
+| pulsar_connection_create_success_count | Gauge | The number of successfully created connections |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:142]:
```suggestion
| pulsar_connection_create_success_count | Gauge | The number of successfully created connections. |
```

File: site2/docs/reference-metrics.md [hunk 6]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+| pulsar_ml_cache_pool_active_allocations_huge | Gauge  | The number of currently active huge allocation in direct arena |
+| pulsar_ml_cache_pool_active_allocations_normal | Gauge  | The number of currently active normal allocations in direct arena |
+| pulsar_ml_cache_pool_active_allocations_small | Gauge  | The number of currently active small allocations in direct arena |
+| pulsar_ml_cache_pool_allocated | Gauge  | The total allocated memory of chunk lists in direct arena |
+| pulsar_ml_cache_pool_used | Gauge  | The total used memory of chunk lists in direct arena |
+| pulsar_ml_cache_used_size | Gauge  | The size in byte used to store the entries payloads |
+| pulsar_ml_count | Gauge  | The number of currently opened managed ledgers |
+| topic_load_times | Summary | The topic load latency calculated in milliseconds |
+| pulsar_active_connections| Gauge | The number of active connections |
+| pulsar_connection_created_total_count | Gauge | The total number of connections |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:141]:
```suggestion
| pulsar_connection_created_total_count | Gauge | The total number of connections. |
```

File: site2/docs/reference-metrics.md [hunk 7]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+| pulsar_ml_cache_pool_active_allocations | Gauge  | The number of currently active allocations in direct arena |
+| pulsar_ml_cache_pool_active_allocations_huge | Gauge  | The number of currently active huge allocation in direct arena |
+| pulsar_ml_cache_pool_active_allocations_normal | Gauge  | The number of currently active normal allocations in direct arena |
+| pulsar_ml_cache_pool_active_allocations_small | Gauge  | The number of currently active small allocations in direct arena |
+| pulsar_ml_cache_pool_allocated | Gauge  | The total allocated memory of chunk lists in direct arena |
+| pulsar_ml_cache_pool_used | Gauge  | The total used memory of chunk lists in direct arena |
+| pulsar_ml_cache_used_size | Gauge  | The size in byte used to store the entries payloads |
+| pulsar_ml_count | Gauge  | The number of currently opened managed ledgers |
+| topic_load_times | Summary | The topic load latency calculated in milliseconds |
+| pulsar_active_connections| Gauge | The number of active connections |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:140]:
```suggestion
| pulsar_active_connections| Gauge | The number of active connections. |
```

File: site2/docs/reference-metrics.md [hunk 8]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+| pulsar_ml_cache_misses_throughput | Gauge  | The amount of data is not retrieved from the cache on the broker side (in byte/s). |
+| pulsar_ml_cache_pool_active_allocations | Gauge  | The number of currently active allocations in direct arena |
+| pulsar_ml_cache_pool_active_allocations_huge | Gauge  | The number of currently active huge allocation in direct arena |
+| pulsar_ml_cache_pool_active_allocations_normal | Gauge  | The number of currently active normal allocations in direct arena |
+| pulsar_ml_cache_pool_active_allocations_small | Gauge  | The number of currently active small allocations in direct arena |
+| pulsar_ml_cache_pool_allocated | Gauge  | The total allocated memory of chunk lists in direct arena |
+| pulsar_ml_cache_pool_used | Gauge  | The total used memory of chunk lists in direct arena |
+| pulsar_ml_cache_used_size | Gauge  | The size in byte used to store the entries payloads |
+| pulsar_ml_count | Gauge  | The number of currently opened managed ledgers |
+| topic_load_times | Summary | The topic load latency calculated in milliseconds |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:139]:
```suggestion
| topic_load_times | Summary | The topic load latency calculated in milliseconds. |
```

File: site2/docs/reference-metrics.md [hunk 9]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+| pulsar_ml_cache_misses_rate | Gauge  | The number of cache misses per second on the broker side. |
+| pulsar_ml_cache_misses_throughput | Gauge  | The amount of data is not retrieved from the cache on the broker side (in byte/s). |
+| pulsar_ml_cache_pool_active_allocations | Gauge  | The number of currently active allocations in direct arena |
+| pulsar_ml_cache_pool_active_allocations_huge | Gauge  | The number of currently active huge allocation in direct arena |
+| pulsar_ml_cache_pool_active_allocations_normal | Gauge  | The number of currently active normal allocations in direct arena |
+| pulsar_ml_cache_pool_active_allocations_small | Gauge  | The number of currently active small allocations in direct arena |
+| pulsar_ml_cache_pool_allocated | Gauge  | The total allocated memory of chunk lists in direct arena |
+| pulsar_ml_cache_pool_used | Gauge  | The total used memory of chunk lists in direct arena |
+| pulsar_ml_cache_used_size | Gauge  | The size in byte used to store the entries payloads |
+| pulsar_ml_count | Gauge  | The number of currently opened managed ledgers |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:138]:
```suggestion
| pulsar_ml_count | Gauge  | The number of currently opened managed ledgers. |
```

File: site2/docs/reference-metrics.md [hunk 10]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+| pulsar_ml_cache_hits_rate | Gauge  | The number of cache hits per second on the broker side. |
+| pulsar_ml_cache_hits_throughput | Gauge  | The amount of data is retrieved from the cache on the broker side (in byte/s). |
+| pulsar_ml_cache_misses_rate | Gauge  | The number of cache misses per second on the broker side. |
+| pulsar_ml_cache_misses_throughput | Gauge  | The amount of data is not retrieved from the cache on the broker side (in byte/s). |
+| pulsar_ml_cache_pool_active_allocations | Gauge  | The number of currently active allocations in direct arena |
+| pulsar_ml_cache_pool_active_allocations_huge | Gauge  | The number of currently active huge allocation in direct arena |
+| pulsar_ml_cache_pool_active_allocations_normal | Gauge  | The number of currently active normal allocations in direct arena |
+| pulsar_ml_cache_pool_active_allocations_small | Gauge  | The number of currently active small allocations in direct arena |
+| pulsar_ml_cache_pool_allocated | Gauge  | The total allocated memory of chunk lists in direct arena |
+| pulsar_ml_cache_pool_used | Gauge  | The total used memory of chunk lists in direct arena |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:136]:
```suggestion
| pulsar_ml_cache_pool_used | Gauge  | The total used memory of chunk lists in direct arena. |
```

File: site2/docs/reference-metrics.md [hunk 11]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+
+| Name | Type   | Description                                          |
+|---|---|---|
+| pulsar_ml_cache_evictions | Gauge  | The number of cache evictions during the last minute. |
+| pulsar_ml_cache_hits_rate | Gauge  | The number of cache hits per second on the broker side. |
+| pulsar_ml_cache_hits_throughput | Gauge  | The amount of data is retrieved from the cache on the broker side (in byte/s). |
+| pulsar_ml_cache_misses_rate | Gauge  | The number of cache misses per second on the broker side. |
+| pulsar_ml_cache_misses_throughput | Gauge  | The amount of data is not retrieved from the cache on the broker side (in byte/s). |
+| pulsar_ml_cache_pool_active_allocations | Gauge  | The number of currently active allocations in direct arena |
+| pulsar_ml_cache_pool_active_allocations_huge | Gauge  | The number of currently active huge allocation in direct arena |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:132]:
```suggestion
| pulsar_ml_cache_pool_active_allocations_huge | Gauge  | The number of currently active huge allocation in direct arena. |
```

File: site2/docs/reference-metrics.md [hunk 12]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+- cluster: cluster=${pulsar_cluster}. ${pulsar_cluster} is the cluster name that you have configured in the `broker.conf` file.
+
+| Name | Type   | Description                                          |
+|---|---|---|
+| pulsar_ml_cache_evictions | Gauge  | The number of cache evictions during the last minute. |
+| pulsar_ml_cache_hits_rate | Gauge  | The number of cache hits per second on the broker side. |
+| pulsar_ml_cache_hits_throughput | Gauge  | The amount of data is retrieved from the cache on the broker side (in byte/s). |
+| pulsar_ml_cache_misses_rate | Gauge  | The number of cache misses per second on the broker side. |
+| pulsar_ml_cache_misses_throughput | Gauge  | The amount of data is not retrieved from the cache on the broker side (in byte/s). |
+| pulsar_ml_cache_pool_active_allocations | Gauge  | The number of currently active allocations in direct arena |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:131]:
```suggestion
| pulsar_ml_cache_pool_active_allocations | Gauge  | The number of currently active allocations in direct arena. |
```

File: site2/docs/reference-metrics.md [hunk 13]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+| pulsar_ml_cache_pool_active_allocations_huge | Gauge  | The number of currently active huge allocation in direct arena |
+| pulsar_ml_cache_pool_active_allocations_normal | Gauge  | The number of currently active normal allocations in direct arena |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:133]:
```suggestion
| pulsar_ml_cache_pool_active_allocations_normal | Gauge  | The number of currently active normal allocations in direct arena. |
```

File: site2/docs/reference-metrics.md [hunk 14]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+| pulsar_ml_cache_pool_active_allocations_small | Gauge  | The number of currently active small allocations in direct arena |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:134]:
```suggestion
| pulsar_ml_cache_pool_active_allocations_small | Gauge  | The number of currently active small allocations in direct arena. |
```

File: site2/docs/reference-metrics.md [hunk 15]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+| pulsar_ml_cache_pool_allocated | Gauge  | The total allocated memory of chunk lists in direct arena |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:135]:
```suggestion
| pulsar_ml_cache_pool_allocated | Gauge  | The total allocated memory of chunk lists in direct arena. |
```

File: site2/docs/reference-metrics.md [hunk 16]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+All the broker metrics are labelled with the following labels:
+- cluster: cluster=${pulsar_cluster}. ${pulsar_cluster} is the cluster name that you have configured in the `broker.conf` file.
+
+| Name | Type   | Description                                          |
+|---|---|---|
+| pulsar_ml_cache_evictions | Gauge  | The number of cache evictions during the last minute. |
+| pulsar_ml_cache_hits_rate | Gauge  | The number of cache hits per second on the broker side. |
+| pulsar_ml_cache_hits_throughput | Gauge  | The amount of data is retrieved from the cache on the broker side (in byte/s). |
+| pulsar_ml_cache_misses_rate | Gauge  | The number of cache misses per second on the broker side. |
+| pulsar_ml_cache_misses_throughput | Gauge  | The amount of data is not retrieved from the cache on the broker side (in byte/s). |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:130]:
```suggestion
| pulsar_ml_cache_misses_throughput | Gauge  | The amount of data (byte per second) that cannot be retrieved from the cache on the broker side. |
```

File: site2/docs/reference-metrics.md [hunk 17]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+### Broker metrics
+All the broker metrics are labelled with the following labels:
+- cluster: cluster=${pulsar_cluster}. ${pulsar_cluster} is the cluster name that you have configured in the `broker.conf` file.
+
+| Name | Type   | Description                                          |
+|---|---|---|
+| pulsar_ml_cache_evictions | Gauge  | The number of cache evictions during the last minute. |
+| pulsar_ml_cache_hits_rate | Gauge  | The number of cache hits per second on the broker side. |
+| pulsar_ml_cache_hits_throughput | Gauge  | The amount of data is retrieved from the cache on the broker side (in byte/s). |
+| pulsar_ml_cache_misses_rate | Gauge  | The number of cache misses per second on the broker side. |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:129]:
```suggestion
| pulsar_ml_cache_misses_rate | Gauge  | The number of cache missed per second on the broker side. |
```

File: site2/docs/reference-metrics.md [hunk 18]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
 
+### Broker metrics
+All the broker metrics are labelled with the following labels:
+- cluster: cluster=${pulsar_cluster}. ${pulsar_cluster} is the cluster name that you have configured in the `broker.conf` file.
+
+| Name | Type   | Description                                          |
+|---|---|---|
+| pulsar_ml_cache_evictions | Gauge  | The number of cache evictions during the last minute. |
+| pulsar_ml_cache_hits_rate | Gauge  | The number of cache hits per second on the broker side. |
+| pulsar_ml_cache_hits_throughput | Gauge  | The amount of data is retrieved from the cache on the broker side (in byte/s). |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:128]:
```suggestion
| pulsar_ml_cache_hits_throughput | Gauge  | The amount of data (byte per second) retrieved from the cache on the broker side. |
```

File: site2/docs/reference-metrics.md [hunk 19]
```diff
@@ -117,6 +117,34 @@ The following metrics are available for broker:
...
+| pulsar_ml_cache_misses_rate | Gauge  | The number of cache misses per second on the broker side. |
+| pulsar_ml_cache_misses_throughput | Gauge  | The amount of data is not retrieved from the cache on the broker side (in byte/s). |
+| pulsar_ml_cache_pool_active_allocations | Gauge  | The number of currently active allocations in direct arena |
+| pulsar_ml_cache_pool_active_allocations_huge | Gauge  | The number of currently active huge allocation in direct arena |
+| pulsar_ml_cache_pool_active_allocations_normal | Gauge  | The number of currently active normal allocations in direct arena |
+| pulsar_ml_cache_pool_active_allocations_small | Gauge  | The number of currently active small allocations in direct arena |
+| pulsar_ml_cache_pool_allocated | Gauge  | The total allocated memory of chunk lists in direct arena |
+| pulsar_ml_cache_pool_used | Gauge  | The total used memory of chunk lists in direct arena |
+| pulsar_ml_cache_used_size | Gauge  | The size in byte used to store the entries payloads |
```
Comment (momo-jun) [inline site2/docs/reference-metrics.md:137]:
```suggestion
| pulsar_ml_cache_used_size | Gauge  | The size used to store the payloads of entries (in bytes). |
```

**Review by Anonymitaet — APPROVED** [review]

**Comment by Anonymitaet** [comment]
@codelipenghui I've renamed your PR based on the PR Naming Guidelines, please follow it, thanks

**Pull request merged**